### DISCUSSION TITLE
Adding bookmark to JSON file

### DISCUSSION THREAD

**Discussion by markolbert** [alefragnani/vscode-bookmarks#638] [Q&A] [answered]
When I try to add a bookmark to a JSON file I get the error message "Open a file first to toggle bookmarks"...which is pretty funny since I'm >>in<< a file.

What am I doing wrong?

**Comment by alefragnani** [comment, marked as answer]
Hi @markolbert ,

You are probably facing a known VS Code limitation, which makes large files invisible to extensions. The first issue was https://github.com/alefragnani/vscode-bookmarks/issues/31.

Unfortunately, there is nothing I can do in the extension to avoid/detect such situation, because VS Code itself does not tell the extensions that an unsupported file is being loaded. So, we will have to wait until VS Code team expand/remove such limitation.

Hope this helps

**Comment by markolbert** [comment]
Thanx for the quick reply. I suspected as much (it's a honking big JSON file, > 57MB). Hopefully the VS Code team will fix the underlying problem soon.
